package.json

```diff
@@ -854,6 +855,28 @@
                     "when": "notebookViewType == 'jupyter-notebook' && jupyter.isnotebooktrusted"
                 }
             ],
+            "notebook/toolbar": [
```

Comment:
What about run all above/below? Those were in the overflow menu. Do we want them in the 'toolbar'?